ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml.cs

```diff
@@ -173,6 +173,20 @@ protected override async Task ValidateAndSave()
             {
                 ModelState.AddModelError("ClinicalDetails.IsMDRTreatment", ValidationMessages.MDRCantChange);
             }
+
+            if ((ClinicalDetails.IsPostMortem ?? false))
+            {
+                if (ClinicalDetails.StartedTreatment ?? false)
+                {
+                    ModelState.AddModelError("ClinicalDetails.IsPostMortem", ValidationMessages.IsPostMortemButStartedTreatment);
+                }
```

Comment:
This validation looks like it ought to happen via attributes on the clinical details model